### PR TITLE
fix: use the right channel_id value in client_info

### PR DESCRIPTION
**Pull request by pjenvey**
also quiet boto3's logging for tests

Closes: #1059

### CODE REVIEW

File: autopush/tests/__init__.py [hunk 1]
```diff
@@ -20,7 +20,8 @@
 
 
 def setUp():
-    logging.getLogger('boto').setLevel(logging.CRITICAL)
+    for name in ('boto', 'boto3', 'botocore'):
+        logging.getLogger(name).setLevel(logging.CRITICAL)
```
Comment (jrconlin) [inline autopush/tests/__init__.py]: crap, missed that. thanks!

**Comment by bbangert** [comment]
Oddly under pypy, Travis spits out:
```
======================================================================

ERROR: test_multiple_delivery_with_multiple_ack

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/home/travis/build/mozilla-services/autopush/.tox/pypy/site-packages/twisted/internet/defer.py", line 653, in _runCallbacks

    current.result = callback(current.result, *args, **kw)

  File "/home/travis/build/mozilla-services/autopush/autopush/websocket.py", line 933, in error_message_overload

    fail.trap(MessageOverloadException)

  File "/home/travis/build/mozilla-services/autopush/.tox/pypy/site-packages/twisted/python/failure.py", line 359, in trap

    self.raiseException()

  File "/home/travis/build/mozilla-services/autopush/.tox/pypy/site-packages/twisted/internet/defer.py", line 653, in _runCallbacks

    current.result = callback(current.result, *args, **kw)

  File "/home/travis/build/mozilla-services/autopush/autopush/websocket.py", line 381, in trap_cancel

    fail.trap(CancelledError)

  File "/home/travis/build/mozilla-services/autopush/.tox/pypy/site-packages/twisted/python/failure.py", line 359, in trap

    self.raiseException()

  File "/home/travis/build/mozilla-services/autopush/.tox/pypy/site-packages/twisted/python/threadpool.py", line 250, in inContext

    result = inContext.theWork()

  File "/home/travis/build/mozilla-services/autopush/.tox/pypy/site-packages/twisted/python/threadpool.py", line 266, in <lambda>

    inContext.theWork = lambda: context.call(ctx, func, *args, **kw)

  File "/home/travis/build/mozilla-services/autopush/.tox/pypy/site-packages/twisted/python/context.py", line 122, in callWithContext

    return self.currentContext().callWithContext(ctx, func, *args, **kw)

  File "/home/travis/build/mozilla-services/autopush/.tox/pypy/site-packages/twisted/python/context.py", line 85, in callWithContext

    return func(*args,**kw)

  File "/opt/python/pypy2.7-5.8.0/lib_pypy/_functools.py", line 45, in __call__

    return self._func(*(self._args + fargs), **fkeywords)

  File "/home/travis/build/mozilla-services/autopush/autopush/db.py", line 379, in wrapper

    return func(self, *args, **kwargs)

  File "/home/travis/build/mozilla-services/autopush/autopush/db.py", line 661, in fetch_timestamp_messages

    Limit=limit

  File "/home/travis/build/mozilla-services/autopush/.tox/pypy/site-packages/boto3/resources/factory.py", line 520, in do_action

    response = action(self, *args, **kwargs)

  File "/home/travis/build/mozilla-services/autopush/.tox/pypy/site-packages/boto3/resources/action.py", line 83, in __call__

    response = getattr(parent.meta.client, operation_name)(**params)

  File "/home/travis/build/mozilla-services/autopush/.tox/pypy/site-packages/botocore/client.py", line 312, in _api_call

    return self._make_api_call(operation_name, kwargs)

  File "/home/travis/build/mozilla-services/autopush/.tox/pypy/site-packages/botocore/client.py", line 601, in _make_api_call

    raise error_class(parsed_response, operation_name)

ClientError: An error occurred (ValidationException) when calling the Query operation: KeyConditionExpressions must only contain one condition per key
```

**Comment by jrconlin** [comment]
@bbangert that's partly due to the swearfest that is boto3's lack of threading. @pjenvey and I are pretty sure that something is corrupting the internal cache that boto3 uses to compile the outbound statement, and that's the most likely candidate.

**Comment by pjenvey** [comment]
100% sure it's due to boto3's session on multiple threads during test_integration

**Pull request merged**